test/Ocelot.UnitTests/Security/IPSecurityPolicyTests.cs

```diff
@@ -335,57 +342,57 @@ private void GivenSetDownstreamRoute()
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string>(), IPBlockedList = new List<string> { "192.168.1.0-192.168.1.10" } });
         }
 
         private void GivenShortRangeAllowedIP()
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string> { "192.168.1.0-10" }, new List<string>(), false));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string> { "192.168.1.0-10" }, IPBlockedList = new List<string>() });
         }
 
         private void GivenShortRangeBlockedIP()
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string>(), new List<string> { "192.168.1.0-10" }, false));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string>(), IPBlockedList = new List<string> { "192.168.1.0-10" } });
         }
 
         private void GivenIpSubnetAllowedIP()
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string> { "192.168.1.0/255.255.255.0" }, new List<string>(), false));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string> { "192.168.1.0/255.255.255.0" }, IPBlockedList = new List<string>() });
         }
 
         private void GivenIpSubnetBlockedIP()
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string>(), new List<string> { "192.168.1.0/255.255.255.0" }, false));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string>(), IPBlockedList = new List<string> { "192.168.1.0/255.255.255.0" } });
         }
 
         private void GivenIpMoreAllowedThanBlocked(bool excludeAllowedInBlocked)
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string> { "192.168.0.0/255.255.0.0" }, new List<string> { "192.168.1.100-200" }, excludeAllowedInBlocked));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string> { "192.168.0.0/255.255.0.0" }, IPBlockedList = new List<string> { "192.168.1.100-200" }, ExcludeAllowedFromBlocked = excludeAllowedInBlocked });
         }
 
         private void GivenIpMoreBlockedThanAllowed(bool excludeAllowedInBlocked)
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string> { "192.168.1.10-20" }, new List<string> { "192.168.1.0/23" }, excludeAllowedInBlocked));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string> { "192.168.1.10-20" }, IPBlockedList = new List<string> { "192.168.1.0/23" }, ExcludeAllowedFromBlocked = excludeAllowedInBlocked });
```

Comment:
Too long C# expression on one line! Impossible to read!
Fix the same formatting issue for all methods/changes above! ☝️

Reply:
Fixed code style